packages/docs/command-line-interface/lockedgold.md

```diff
@@ -0,0 +1,139 @@
+---
+description: View information about all of the account's commitments
+---
+
+## Commands
+
+### List
+
+View information about all of the account's commitments
+
+```
+USAGE
+  $ celocli lockedgold:list ACCOUNT
+
+EXAMPLE
+  list 0x5409ed021d9299bf6814279a6a1411a7e866a631
+```
+
+_See code: [packages/cli/src/src/commands/lockedgold/list.ts](https://github.com/celo-org/celo-monorepo/tree/master/packages/cli/src/src/commands/lockedgold/list.ts)_
+
+### Lockup
+
+Create a locked Gold commitment given notice period and gold amount
+
+```
+USAGE
+  $ celocli lockedgold:lockup
+
+OPTIONS
+  --from=from                  (required)
+  --goldAmount=goldAmount      (required) unit amount of gold token (cGLD)
```

Comment:
Another inconsistency - "gold" is lowercase. I guess this might be repeated elsewhere in the docs (and CLI source code). Maybe out of scope of this PR to standardize everywhere?

Reply:
out of scope I think